### PR TITLE
Enhance issues statistic

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
fetch: multiple urls per day
android: show everything in stacked diagram

image

### CODE REVIEW

File: translations/status/fetchStats.php [hunk 1]
```diff
@@ -2,48 +2,55 @@
 
 $configs = [
 	[
-		'url' => 'https://github.com/nextcloud/server/issues?q=is%3Aissue%20is%3Aopen%20-label%3Aenhancement%20-label%3Aspec%20-label%3Asecurity%20-label%3Abug%20-label%3Apapercut%20-label%3Aoverview%20%20-label%3A%22technical%20debt%22%20',
+	    'url' => array(
```
Comment (MorrisJobke) [inline translations/status/fetchStats.php]: please use `[` instead of `array(` and `]` instead of the closing `)`.

Comment (MorrisJobke) [reply translations/status/fetchStats.php]: And the indentation is off :'(

File: translations/status/fetchStats.php [hunk 2]
```diff
@@ -2,48 +2,55 @@
...
-
-	$result = preg_replace('!<.*>!U', '', $matches[0][0]);
-
-	preg_match('!(\d+) open!i', $result, $matches);
-
-	if (!isset($matches[1])) {
-	    die('Could not find result set.');
-	}
-	$number = $matches[1];
+    $number = "";
```
Comment (MorrisJobke) [inline translations/status/fetchStats.php]: Indentation ...

**Comment by tobiasKaminsky** [comment]
@MorrisJobke I updated the PR according to your remarks

**Pull request merged**